apps/prairielearn/src/pages/studentAssessmentInstance/studentAssessmentInstance.html.ts

```diff
@@ -0,0 +1,751 @@
+          ...resLocals,
+          navPage: 'assessment_instance',
+        })}
+        <main id="content" class="container">
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">
+              ${resLocals.assessment_set.abbreviation}${resLocals.assessment.number}:
+              ${resLocals.assessment.title}
+              ${resLocals.assessment.group_work ? html`<i class="fas fa-users"></i>` : ''}
+            </div>
+
+            <div class="card-body">
+              ${resLocals.assessment.allow_real_time_grading && resLocals.assessment_instance.open
+                ? html`
+                    <div class="alert alert-warning">
+                      This assessment will only be graded after it is finished. You should save
+                      answers for all questions and your exam will be graded later. You can use the
+                      <span class="badge badge-outline badge-light">Finish assessment</span> button
+                      below to finish and calculate your final grade.
+                    </div>
+                  `
+                : ''}
+              <div class="row align-items-center">
+                ${!resLocals.assessment.allow_real_time_grading &&
+                resLocals.assessment_instance.open
+                  ? html`
+                      <div class="col-md-3 col-sm-12">
+                        Total points: ${formatPoints(resLocals.assessment_instance.max_points)}
+                        ${resLocals.assessment_instance.max_bonus_points
+                          ? html`
+                              <br />
+                              (${resLocals.assessment_instance.max_bonus_points} bonus
+                              ${resLocals.assessment_instance.max_bonus_points > 1
+                                ? 'points'
+                                : 'point'}
+                              possible)
+                            `
+                          : ''}
+                      </div>
+                      <div class="col-md-9 col-sm-12">
+                        ${resLocals.assessment_instance.open && resLocals.authz_result.active
+                          ? html`
+                              Assessment is <strong>open</strong> and you can answer questions.
+                              <br />
+                              Available credit: ${resLocals.authz_result.credit_date_string}
+                              ${StudentAccessRulesPopover({
+                                accessRules: resLocals.authz_result.access_rules,
+                                assessmentSetName: resLocals.assessment_set.name,
+                                assessmentNumber: resLocals.assessment.number,
+                              })}
+                            `
+                          : html`
+                              Assessment is <strong>closed</strong> and you cannot answer questions.
+                            `}
```

Comment:
Nit: this is repeated twice. Separate component?

Reply:
Done in 49bdc36.